### ISSUE TITLE
如何设置参数可以使用预训练的某一层得到输出

### ISSUE THREAD

**Issue by IsDling** [QwenLM/Qwen3-VL#1436]
目的是这样的，我通过一个encoder将图像信息提取为[batch, seq_len,dim]的tensor，现在通过下面代码想要将输入送入到预训练的vision encoder 的第27层，得到输出，请问cu_seqlens和position_embeddings参数如何设置？

batch_size = 24
seq_len = 10
dim = 1280

model = Qwen2_5_VLForConditionalGeneration.from_pretrained(
    "models/Qwen/Qwen2.5-VL-3B-Instruct-AWQ", torch_dtype=torch.float32, device_map="auto"
)

used_model = model.visual.blocks[27].cuda()
input = torch.zeros((batch_size, seq_len, dim)).cuda()
x_llm = used_model(hidden_states=input, cu_seqlens=, position_embeddings=)[0]
print(x_llm.shape)
